src/app/modules/gui.cpp

```diff
@@ -450,13 +453,19 @@ void save_window_pos(Window* window, const char* section)
   if (!window->lastNativeFrame().isEmpty()) {
     const os::Window* mainNativeWindow = manager->display()->nativeWindow();
     rc = window->lastNativeFrame();
+    // De-scale the window size to save it normalized.
+    rc.w /= guiscale();
+    rc.h /= guiscale();
     set_config_rect(section, "WindowFrame", rc);
     rc.offset(-mainNativeWindow->frame().origin());
     rc /= mainNativeWindow->scale();
   }
   else {
     del_config_value(section, "WindowFrame");
     rc = window->bounds();
+    // De-scale the window size to save it normalized.
+    rc.w /= guiscale();
+    rc.h /= guiscale();
   }
 
   set_config_rect(section, "WindowPos", rc);
```

Comment:
The WindowPos keeps the same position on the screen. This may be fine, but I feel like it's not correct in my opinion.
Example:
- Guiscale 200%, go to File > Export > Export Sprite Sheet, suppose the dialog is centered on the screen. Close the dialog.
- Change the guiscale to 100%, open again the Export Sprite Sheet dialog.
- Result: The window will appear to be shifted to the top/left. When it would have been desirable to find it centered on the screen.
For this reason, I am going to propose to save the unscaled central position of the dialog, to be able to use it and re-scale it as desired guiscale (of course limiting the top/left corner of the window inside the editor).